controller/authenticate/twofactor.js

```diff
@@ -24,27 +19,14 @@ const exampleAppConfig = require('../../config');
  * @param phoneNumber a phone number to send an SMS with the 2fa code
  */
 const sendSmsCode = async (phoneNumber) => {
-    const text2fa = {
-        Text2FA: [{
-            AccountId: [exampleAppConfig.accountId]
-        }, {
-            ApplicationId: [exampleAppConfig.applicationId]
-        }, {
-            Action: ["testing"]
-        }, {
-            To: [{
-                PhoneNum: [phoneNumber]
-            }]
-        }, {
-            From: [{
-                PhoneNum: [exampleAppConfig.phoneNum]
-            }]
-        }]
-    };
-
     await axios.post(
-        exampleAppConfig.twofactorUrl,
-        text2fa,
+        exampleAppConfig.twofactorUrl(exampleAppConfig.accountId, "messaging"),
+        {
+            to: phoneNumber,
+            from: exampleAppConfig.phoneNum,
+            applicationId: exampleAppConfig.applicationId,
+            scope: "Two-Factor"
```

Comment:
typically we use "testing" as the value here, but we also use "testing" as the appName often, resulting in a weird voice msg where voice engine speaks "your ... testing testing code is ..." which sounds like we repeated a word in our coded string, but I don't like the machine speaking "Two-Factor", (will it get that right?  why is it capitalized?  I know that we are just supporting messaging here, but we or they will extend this app to support voice easily)

I suggest we use the word "example" since this is an example app.  I can fix the python app too.

Reply:
`example` sounds fine to me, matches the description of the application more than just `testing`.